_layouts/about.html

```diff
@@ -2,52 +2,57 @@
 layout: default
 ---
 
-<div class="post">
-
-  <header class="post-header">
-    <h1 class="post-title">
-     {% if site.title == "blank" %}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }} {{ site.last_name }}{% else %}{{ site.title }}{% endif %}
-    </h1>
-     <p class="desc">{{ page.description }}</p>
-  </header>
-
-  <article>
-    {% if page.profile %}
-    <div class="profile float-{% if page.profile.align == 'left' %}left{% else %}right{% endif %}">
-      {% if page.profile.image %}
-        {% assign profile_image_path = page.profile.image | prepend: 'assets/img/' %}
-        {% include figure.html 
-          path=profile_image_path 
-          class="img-fluid z-dept-1 rounded"%}
-      {% endif %}
-      {% if page.profile.address %}
-        <div class="address">
-          {{ page.profile.address }}
-        </div>
-      {% endif %}
-    </div>
-    {% endif %}
-
-    <div class="clearfix">
-      {{ content }}
-    </div>
-
-    {% if page.news %}
-      {% include news.html %}
-    {% endif %}
-
-    {% if page.selected_papers %}
-      {% include selected_papers.html %}
-    {% endif %}
-
-    {% if page.social %}
-    <div class="social">
-      <div class="contact-icons">
-        {% include social.html %}
-      </div>
-      <div class="contact-note">{{ site.contact_note }}</div>
-    </div>
-    {% endif %}
-  </article>
+<!-- about.html -->
+      <div class="post">
+        <header class="post-header">
+          <h1 class="post-title">
+           {% if site.title == "blank" -%}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }} {{ site.last_name }}{%- else -%}{{ site.title }}{%- endif %}
+          </h1>
+          <p class="desc">{%- if page.subtitle -%}{{ page.subtitle }}{%- else -%}{{ page.description }}{%- endif -%}</p>
```

Comment:
```suggestion
          <p class="desc">{{ page.subtitle }}</p>
```
let's remove fall back to `page.description` here. about page shouldn't have description field specified, so that the meta tag uses `site.description` for that page. you already have 
fall back to site description implemented in `_includes/metadata.html`:
https://github.com/alshedivat/al-folio/blob/964b0eb741054614d5f9cd3cef4fea3532bb44a3/_includes/metadata.html#L28